libs/crossplane/config.jsonnet

```diff
@@ -85,9 +85,9 @@ config.new(
       localName: 'crossplane_jet_mongodbatlas',
     },
     {
-      output: 'provider-nop/0.1.1',
+      output: 'provider-nop/0.2.0',
       prefix: '^io\\.crossplane\\.nop\\..*',
-      crds: ['https://doc.crds.dev/raw/github.com/crossplane-contrib/provider-nop@v0.1.1'],
+      crds: ['https://doc.crds.dev/raw/github.com/crossplane-contrib/provider-nop@v0.2.0'],
       localName: 'crossplane_nop',
     },
```

Comment:
You might want to add a new entry when there are breaking changes between v0.1.1 and v0.2.0